nems/util/shr_flux_mod.F90

```diff
@@ -315,7 +315,11 @@ real(R8)    :: psix2  ! stability function at ztref reference height
              !(1.0_R8-stable) * chxcdu + stable * chxcds
 
              !--- shift all coeffs to measurement height and stability ---
-             rd = rdn / (1.0_R8 + rdn/loc_karman*(alz-psimh))
+             if (ocn_surface_flux_scheme == -1)then
+               rd = rdn
```

Comment:
Do we have any documentation to support the "rd=rdn"  here?

Reply:
This needs to be provided by @hyunchul386. I've tried to implement his NEMS bulk flux branch.